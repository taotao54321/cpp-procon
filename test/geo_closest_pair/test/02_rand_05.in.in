1000
-0.999487 -0.387958
0.498792 0.611359
-0.495417 -0.398019
0.775244 0.595147
-0.467190 -0.345072
0.079323 -0.347990
-0.991305 -0.643643
0.754306 0.280497
0.182811 0.237701
-0.225807 0.512123
0.531520 0.636436
-0.407931 -0.577307
-0.921011 0.914139
-0.056021 0.886449
0.116717 0.311941
-0.974492 0.633587
-0.989412 0.172707
0.496058 0.575887
-0.937199 -0.738450
-0.687100 -0.709556
-0.825533 -0.343589
0.364277 -0.672002
0.958236 0.178441
0.427958 0.825224
-0.148085 0.206610
0.637535 0.614151
0.250013 0.912186
0.983337 -0.635522
-0.639414 -0.456683
0.763648 -0.726545
-0.940953 0.665794
0.482588 0.799647
0.788214 0.572017
0.861809 -0.985957
-0.330402 -0.411625
0.820529 0.405573
-0.438141 -0.525338
0.590419 0.898372
-0.927658 -0.407696
-0.863361 0.155914
0.247991 0.885625
0.571409 -0.538141
0.317105 0.060745
-0.978725 -0.773125
-0.163944 0.706050
-0.042428 0.048424
0.378967 0.556344
0.302076 -0.125238
-0.403009 0.411669
0.441822 -0.565545
-0.913321 -0.499107
-0.912332 0.262647
-0.930183 -0.844134
0.127541 0.139978
0.896466 -0.404644
-0.015453 0.372090
-0.697892 -0.192714
-0.130721 0.809018
-0.050549 -0.295068
0.352767 -0.014445
0.224295 -0.868607
-0.799836 -0.919651
-0.893077 -0.657328
-0.636458 0.630611
0.152596 0.940950
-0.884210 0.694321
0.987637 0.961895
0.918406 -0.458307
-0.879171 0.685419
-0.877295 -0.090986
-0.594858 -0.795724
-0.804313 0.575890
0.870562 -0.680784
0.875375 0.486412
0.454824 0.473796
-0.676545 0.779643
-0.463745 -0.641372
0.276534 -0.962191
0.819580 -0.818557
0.372060 0.272257
-0.890223 0.024614
-0.300608 0.274605
-0.850279 0.873513
-0.836409 -0.673159
0.252267 -0.764374
-0.843124 -0.222974
-0.419409 0.152635
0.966357 0.090842
-0.837719 0.196784
-0.981425 0.545405
-0.324675 -0.725133
-0.834129 0.407912
-0.831002 0.344104
0.043362 0.590671
-0.121204 0.575172
0.927265 -0.105258
0.097902 -0.719317
-0.331255 -0.511378
-0.091888 0.801117
-0.723969 -0.372608
0.996302 -0.651219
-0.875828 0.003261
0.028730 0.068143
-0.009997 0.715389
-0.792795 0.210943
0.812238 0.055693
-0.760845 -0.676713
0.716803 0.812323
0.572045 0.556505
0.641756 0.392723
-0.667029 0.165441
-0.297665 -0.692914
-0.736046 0.536886
0.500474 -0.715014
-0.374715 0.725004
0.546282 0.821882
-0.138923 0.792150
-0.534907 -0.821879
-0.079149 0.038373
-0.760133 0.196120
0.846867 0.088651
-0.978276 0.773862
-0.464175 0.254700
-0.699947 0.705706
-0.785129 0.870943
0.557445 0.257368
-0.746198 0.648827
0.975283 0.669134
0.082681 -0.552706
-0.752253 0.836090
0.339014 -0.300642
-0.365696 0.014280
-0.105969 -0.609867
0.898499 0.177529
0.610513 -0.825132
-0.926361 0.803644
-0.726492 -0.700692
0.391924 -0.458993
0.698009 -0.760359
-0.364732 0.530276
-0.770262 0.352739
0.124283 -0.100772
-0.255467 -0.678102
-0.287513 0.261130
0.687829 0.284004
-0.455972 0.684871
-0.819543 0.801345
-0.486164 0.430247
-0.778219 -0.362276
0.042010 -0.901759
-0.575749 -0.933375
0.709325 -0.100380
-0.893129 -0.626824
-0.442513 0.175694
-0.720307 0.963971
-0.174541 -0.756696
-0.708468 0.858359
-0.685241 0.369301
0.581282 -0.852552
0.397786 0.846927
0.804955 -0.916856
0.818873 0.280149
0.635629 0.859165
0.186345 -0.460700
0.123126 -0.722669
-0.494006 -0.354278
0.823733 0.777672
0.198058 -0.359460
0.801164 0.032028
0.080645 -0.706787
-0.672798 -0.467157
-0.014686 0.446487
-0.579373 0.261789
0.375069 -0.771440
0.148856 -0.882680
-0.663597 0.394940
-0.528802 -0.931331
0.442115 -0.637114
-0.115225 0.006708
-0.059582 -0.821710
-0.897244 0.999808
-0.498887 -0.542273
-0.628858 0.168421
-0.497243 -0.999487
0.978734 -0.201288
-0.648623 0.735347
0.989166 -0.462630
0.740283 -0.201364
-0.925957 -0.648620
-0.644540 0.375991
0.787444 -0.629712
-0.638935 0.350927
0.318576 -0.845601
0.690862 -0.710837
-0.008444 0.246443
-0.368454 0.070243
0.774181 0.579999
-0.247617 0.416740
0.514470 0.571923
0.454445 -0.897393
-0.450016 -0.036286
-0.553389 -0.361146
-0.464569 -0.469723
-0.614430 0.712243
0.780668 0.068467
-0.604693 -0.182479
0.034234 0.616664
0.011684 -0.106617
-0.161308 0.237445
-0.347819 -0.233065
0.792820 -0.676999
-0.615260 -0.308018
-0.418851 -0.408920
0.508834 0.283211
0.218050 -0.343714
-0.619714 -0.787624
0.543429 0.022549
0.189707 0.638402
-0.244012 0.629172
0.706986 0.530457
0.657833 0.767375
-0.783858 -0.548775
-0.412618 -0.024309
0.677661 0.668319
-0.268751 -0.642116
-0.576597 0.083096
0.612257 -0.085763
0.920122 0.914355
-0.470635 -0.781328
-0.616874 -0.830636
0.044181 -0.209728
-0.565376 -0.966381
-0.488257 -0.621164
-0.562253 -0.225118
-0.937887 0.504038
0.747795 -0.593241
0.410643 -0.231525
-0.370459 0.705247
-0.353714 0.631816
-0.713615 0.401533
0.432921 0.917692
-0.550164 -0.406561
-0.530753 -0.012237
-0.780702 -0.472880
0.936848 0.893855
0.293998 0.322607
-0.281085 -0.687843
0.499575 -0.114275
-0.563553 0.755250
-0.882899 -0.831156
-0.125182 0.197920
-0.371104 -0.214842
-0.537542 0.921086
-0.381565 -0.237557
-0.296621 -0.073915
-0.094875 0.837258
-0.581157 0.495444
-0.542708 0.113214
-0.237555 -0.461384
0.657098 0.058140
0.632722 0.051315
0.279018 0.228243
-0.420293 0.759593
0.410756 -0.599312
-0.271980 -0.286605
0.414691 0.764397
-0.508687 -0.322552
-0.676977 0.852340
-0.778602 -0.267690
0.727833 -0.754954
-0.375406 0.584023
0.769832 -0.020708
-0.495142 0.555741
0.979116 -0.826877
0.739677 -0.528278
0.335202 -0.933077
-0.982238 -0.983910
0.980250 -0.017490
-0.908350 -0.327140
0.124062 0.073606
-0.576789 -0.860909
-0.510843 -0.227135
0.293074 0.806748
-0.481239 -0.505840
0.332355 -0.420145
-0.518108 -0.181652
0.508892 -0.937161
-0.150859 0.356480
0.991724 -0.960296
-0.469307 -0.156794
-0.749506 0.082140
-0.467918 -0.734477
-0.652619 0.710175
0.653713 -0.613329
-0.998968 0.092258
-0.397372 -0.084540
-0.188631 -0.564813
-0.672273 -0.651974
0.375204 0.403113
0.315283 -0.060094
0.187860 0.880253
-0.123015 0.413754
-0.653979 0.874713
-0.447461 -0.030584
-0.766115 -0.799467
-0.651597 -0.272963
0.805633 -0.087645
-0.442105 0.764475
-0.751620 0.315226
0.231873 0.080113
-0.316931 -0.734806
0.610270 -0.886115
0.685105 0.354168
-0.022146 0.525687
-0.691848 -0.700192
0.175281 0.479908
-0.525214 -0.140921
-0.495020 0.185965
-0.376979 -0.777888
-0.932989 0.339339
-0.459375 -0.696084
-0.998471 -0.650920
-0.256022 -0.096975
0.715432 0.040500
0.029913 -0.134850
0.258562 -0.263322
-0.916667 0.808177
-0.401051 -0.846403
-0.542738 -0.867587
-0.525508 0.263945
-0.115497 -0.968140
-0.396834 -0.251626
-0.323394 0.848175
0.749460 0.058139
0.242139 -0.494628
0.100435 -0.895424
0.474706 -0.446984
-0.816735 -0.819031
-0.685932 0.032084
-0.484557 -0.739065
-0.272985 0.223283
-0.228287 -0.360144
-0.971839 -0.224603
-0.054976 0.540397
-0.112821 0.417099
-0.534871 -0.697893
0.776809 -0.341903
0.577212 -0.379028
0.047899 -0.991187
-0.063793 -0.255732
-0.153518 -0.895179
0.916595 -0.048597
-0.981057 0.125605
-0.063013 -0.108524
0.122400 -0.295675
-0.552889 0.871574
0.801438 0.786476
-0.349738 -0.611178
-0.021728 0.463466
0.961545 -0.171330
-0.363538 0.702329
0.274932 -0.599539
-0.023619 0.733416
0.485828 -0.678688
0.968078 -0.668200
-0.356336 -0.182119
-0.327044 0.236604
0.593946 0.356508
-0.343069 0.225329
0.582650 0.949528
0.580307 0.060754
-0.968750 0.611314
0.124592 0.889428
-0.635130 0.043203
-0.315193 0.071346
-0.313505 -0.180048
-0.224671 0.111745
0.632172 -0.101112
-0.307175 0.843662
-0.361552 0.209765
-0.270473 0.458591
0.689969 -0.827128
0.448198 -0.075716
-0.704593 -0.732612
0.307171 -0.718230
-0.764548 0.486574
-0.295105 -0.435907
0.579665 -0.657597
-0.281224 0.110509
-0.291163 0.080850
-0.247996 -0.463066
0.897587 -0.130808
-0.293209 0.671438
-0.946843 0.242272
-0.278137 0.922526
-0.224903 -0.764636
-0.959186 0.942106
-0.589370 0.207947
0.718234 -0.678657
-0.511116 0.277301
-0.384725 0.751387
0.650701 -0.474739
-0.856886 0.359053
0.436774 0.127868
-0.262564 0.380309
-0.301429 0.146608
0.987861 0.274588
-0.866785 -0.171865
-0.408969 0.111909
0.121891 -0.337025
-0.253507 -0.278235
-0.248080 -0.407305
0.896959 0.368666
-0.621675 -0.159313
-0.587053 -0.358937
0.868728 -0.412237
0.451098 0.115625
0.250807 0.201559
0.446623 -0.277036
-0.145109 -0.800238
-0.228475 0.195730
0.164395 0.615832
0.907780 0.798558
-0.364732 -0.354513
-0.225584 -0.553725
0.490555 -0.184056
0.342552 0.102941
-0.047517 -0.326390
0.191786 0.850137
0.072303 0.239382
0.769934 0.177312
0.494231 0.266705
-0.303074 -0.437796
0.190399 -0.767181
-0.209381 0.755359
-0.408231 -0.645740
-0.067820 0.721425
-0.387855 -0.448248
0.230299 -0.949013
-0.433531 0.309938
0.616864 -0.542335
-0.544180 0.824787
-0.169872 -0.808713
0.479756 0.076611
0.050329 -0.170891
0.601804 0.819268
-0.197260 0.921500
0.623212 0.453874
0.045234 -0.332989
-0.166529 -0.839032
-0.323619 0.486542
-0.073931 -0.106028
-0.159372 -0.122671
-0.157284 0.820512
0.270149 -0.141531
-0.907540 -0.424684
-0.152372 -0.562286
0.501003 0.137299
0.296294 -0.327620
0.725615 0.609872
-0.721572 -0.274192
0.615783 -0.306722
-0.177240 -0.663451
0.980511 0.123011
0.510709 -0.328974
0.494924 0.769303
-0.175723 0.415221
-0.539778 0.083761
0.306346 0.260959
0.846028 0.749280
-0.141705 -0.325185
0.502492 0.127153
0.264100 -0.924371
0.593039 0.674515
-0.409881 0.668416
-0.046906 0.029841
-0.846192 -0.771197
-0.665664 -0.300948
-0.900321 -0.606617
-0.699615 0.978642
-0.675225 0.067501
0.496028 0.287209
0.880698 0.821948
-0.917194 -0.338362
-0.920168 -0.590541
-0.738331 -0.644251
-0.506139 0.095567
0.984199 -0.024451
-0.843329 -0.191487
-0.667661 -0.776613
-0.332402 0.314050
0.836747 0.814693
0.119393 -0.152768
-0.170337 -0.858218
-0.737806 -0.793052
-0.796981 0.766764
-0.918468 0.941991
0.536093 0.842463
-0.062597 0.103624
-0.059155 -0.049807
-0.786438 -0.006951
-0.395245 0.688171
-0.305129 -0.510057
0.868822 0.243827
-0.549549 -0.668487
0.472536 -0.171007
0.887588 -0.506806
0.120373 0.391832
0.645843 0.656336
-0.026609 0.144559
-0.220815 0.048284
0.502116 -0.010137
0.366505 -0.046665
-0.096939 -0.662043
0.195395 0.247330
0.835268 0.328075
-0.617800 -0.262124
0.611052 -0.818791
-0.011996 0.320350
0.709700 -0.255128
-0.552912 0.105440
0.400797 0.634063
0.571785 -0.203529
-0.915821 0.031048
-0.993019 0.583078
-0.585563 0.854180
-0.537471 0.545577
-0.278081 -0.243994
0.017588 0.873084
0.646226 -0.939308
-0.903310 0.916380
-0.170434 -0.947148
-0.911126 0.007073
0.347808 0.579824
-0.716277 -0.466500
-0.644795 -0.749341
-0.022436 -0.378593
0.970695 -0.734395
0.764775 -0.525998
0.391493 -0.444003
-0.872958 -0.869504
0.057803 -0.946151
0.066916 0.019028
-0.460571 0.250947
-0.598305 0.070655
-0.473627 -0.559442
-0.722267 0.841773
0.729288 0.925028
0.081112 0.324449
-0.905371 0.562861
0.905335 -0.060146
0.084059 -0.537761
-0.261685 -0.195474
-0.053032 0.989097
-0.375622 0.039952
0.096869 -0.536556
0.097697 0.116576
0.898508 -0.391153
-0.443424 -0.947273
-0.386346 0.464548
-0.863858 0.454957
0.735275 0.418438
0.474579 -0.357387
0.787802 -0.615429
0.114643 -0.871612
0.387158 0.215462
-0.313437 0.487821
-0.109367 0.285742
-0.407935 -0.995516
-0.741510 0.565920
-0.050999 0.020757
0.819255 -0.368919
-0.585021 0.416838
0.154088 0.473120
0.859912 0.038501
-0.145647 -0.207979
-0.453130 0.924205
0.655522 0.458985
0.137100 0.505477
0.631816 -0.683791
0.144206 0.687919
0.301784 0.231288
0.795604 -0.137673
-0.431083 0.949702
0.277941 0.402429
0.531573 0.135434
0.477141 0.783203
0.647593 -0.244033
-0.235897 0.512807
0.164670 0.576008
-0.395722 -0.936633
0.297626 0.750959
0.170475 -0.921397
0.171472 0.137296
0.549308 0.748198
0.174279 0.175801
0.856277 0.476715
-0.044641 -0.313348
0.311771 -0.158349
0.181697 -0.234689
-0.604198 -0.461791
-0.840393 -0.032309
-0.568393 0.827213
-0.434650 -0.691907
-0.879739 -0.492999
-0.930421 0.844831
-0.335926 0.302655
-0.540587 0.431661
-0.742700 -0.972763
-0.103420 -0.366756
0.204798 -0.506851
0.466576 -0.293313
0.220705 -0.633171
-0.268506 0.190312
-0.154303 -0.090305
0.087167 -0.189062
-0.533152 -0.910130
0.055489 0.413494
0.331545 0.483969
0.333162 -0.917283
-0.603034 -0.425136
-0.421447 -0.518315
0.609898 -0.587758
0.359976 -0.452127
0.585609 -0.330460
-0.048729 0.394300
0.391376 -0.694372
0.446939 0.517066
-0.361468 -0.091483
0.951753 0.628323
0.174457 0.690196
0.260708 -0.651023
-0.894488 -0.309104
-0.800124 0.762395
0.265516 -0.746461
0.828231 0.454732
0.271885 0.838208
0.147683 0.177734
-0.426457 -0.793626
0.433817 -0.415237
-0.020979 0.009185
0.240501 0.345949
0.280845 0.184126
0.758795 -0.033770
-0.654478 0.769328
-0.543117 0.037624
-0.399671 0.538983
-0.529566 -0.990942
0.432815 -0.553839
-0.117605 0.920749
0.303499 0.752365
-0.540822 0.444792
-0.133687 0.995379
0.474027 0.877625
0.781633 0.815524
0.961559 0.949442
0.310069 -0.310686
0.934292 0.573754
-0.538655 0.160383
0.274199 0.046948
-0.681832 -0.162629
-0.897942 -0.415744
0.328953 0.514809
-0.580860 -0.215561
0.090409 0.718190
-0.679579 0.277959
0.959311 -0.844486
-0.480753 0.970720
-0.100421 0.599785
0.101773 -0.842812
0.338489 -0.666590
-0.604236 -0.315178
-0.681761 -0.076524
0.226874 -0.382514
-0.561409 0.121680
-0.918367 0.613451
0.349522 -0.051463
0.673683 -0.907811
-0.645598 -0.840747
0.932871 0.174250
0.016734 0.545784
0.078340 0.339120
0.855837 0.757201
0.780249 -0.782938
-0.586051 0.599923
0.494475 -0.019293
0.306496 0.953261
0.376917 -0.136345
-0.474350 0.484842
0.382642 0.583691
-0.775415 0.019397
0.389704 -0.746863
0.391092 -0.370998
0.250831 -0.983730
0.743431 -0.130517
0.391900 -0.020193
-0.568801 0.120183
0.528150 0.047540
-0.824104 0.760604
-0.108502 -0.262424
0.398067 -0.935257
0.816704 0.306946
-0.295099 0.731356
0.407946 0.604894
0.395116 0.274412
-0.922398 0.539533
-0.550267 0.924737
-0.322612 -0.036776
0.426620 -0.651016
0.427094 -0.918189
0.427422 0.333238
0.330285 -0.505646
-0.681999 0.129821
0.913420 0.372221
-0.497247 0.165066
0.259082 -0.516929
0.440080 0.206961
0.440497 -0.234654
-0.182724 0.877367
-0.751049 -0.217499
0.883606 0.816265
-0.759941 -0.164073
0.445171 0.149224
-0.744358 -0.952590
-0.323136 0.710806
0.936700 -0.546097
0.448360 0.983244
-0.631795 -0.351182
-0.689285 -0.425148
-0.815634 0.464517
0.399150 0.898956
0.224598 -0.191221
0.913665 0.891610
0.941178 0.892054
0.464420 -0.844853
-0.456677 -0.708129
0.441699 -0.792955
0.179325 -0.922669
0.937399 0.377475
-0.807513 0.524923
0.573761 0.958062
0.016273 -0.611483
-0.594697 -0.658077
0.746404 0.935019
-0.767008 0.445917
0.809296 0.020758
0.131543 0.251018
-0.445185 0.544518
0.952051 0.712453
-0.683317 -0.809584
-0.594792 0.812189
0.755649 0.262769
-0.177509 -0.863885
0.608779 -0.586889
0.146118 0.680951
0.832304 -0.207581
0.240117 0.693463
-0.943201 -0.437729
0.683130 -0.970070
0.842762 -0.592854
0.033833 -0.286777
-0.496532 -0.487461
0.739196 0.270686
0.852121 0.484387
0.509169 -0.323255
0.240376 -0.700143
0.942314 0.679486
0.447017 0.925923
0.515048 0.558710
0.701651 0.751529
0.529348 0.545053
0.101488 -0.343375
-0.021319 -0.506955
-0.343151 0.352739
0.533307 -0.360121
-0.650234 -0.119455
-0.247131 0.844432
0.544819 0.913666
-0.947305 -0.320184
-0.182483 0.678246
0.903921 0.255225
0.550855 0.936717
0.553787 -0.508302
-0.841445 -0.267326
-0.949661 0.556632
0.564809 0.671831
-0.213323 -0.156776
-0.070607 -0.624652
-0.769079 0.306935
-0.544859 0.523410
-0.490037 0.424080
-0.740126 0.008453
0.418065 -0.853010
0.114577 0.524510
0.936240 -0.425774
0.330376 0.475852
0.010903 -0.429829
0.585815 0.727678
-0.264471 0.862947
-0.680884 -0.142745
-0.688940 -0.404333
-0.943669 0.130198
-0.694199 0.491092
0.855274 -0.656270
-0.226408 0.308107
0.637638 0.777867
-0.411104 -0.667662
0.457029 -0.830528
-0.083101 0.246443
-0.214126 0.617767
-0.581842 -0.323020
-0.221704 -0.418884
-0.198025 0.010604
-0.318342 0.540146
0.491977 0.969998
-0.380133 -0.112776
0.631751 0.591794
-0.428385 0.507088
-0.638513 0.730633
-0.656775 -0.774216
0.259880 -0.134473
-0.778045 0.237821
-0.686158 0.793359
-0.869392 -0.653896
-0.501096 -0.102942
-0.673688 -0.429615
-0.210262 -0.573374
-0.309603 0.667743
-0.872877 -0.496543
0.857954 0.412729
0.605465 0.597442
0.136706 -0.861167
-0.653905 0.327815
-0.399731 0.995410
-0.651869 -0.880146
0.442493 -0.517670
-0.871906 -0.377957
0.332999 -0.320928
-0.134263 -0.776673
0.687221 -0.853328
0.095908 -0.247411
0.688367 0.364569
0.346833 0.986256
-0.930249 0.459825
0.912042 0.968567
0.912851 0.460237
-0.048842 -0.164186
0.700956 -0.363885
0.013161 -0.121946
0.705731 -0.845151
0.748180 0.114775
-0.837203 -0.596843
0.368992 0.847000
0.774935 0.824031
-0.242796 -0.971557
0.615763 0.736554
0.713840 0.883623
-0.113519 0.742842
-0.658979 -0.675937
0.694543 0.403983
0.639985 0.139975
0.721672 0.401429
-0.991354 0.822317
0.014213 0.238859
0.547280 -0.542410
0.153966 -0.139354
0.782304 -0.137829
-0.647025 0.747377
-0.301002 -0.287160
0.914876 -0.580805
0.460963 -0.284064
0.741166 0.180893
-0.721347 -0.065035
0.003740 -0.411145
0.648355 -0.616404
0.716507 0.158558
0.708927 -0.088785
-0.890380 -0.294774
-0.118936 0.767854
-0.673634 0.485560
-0.856061 -0.178059
0.761281 0.662998
-0.484586 -0.509749
0.894332 -0.227175
-0.943380 -0.806496
-0.256349 -0.260269
-0.217516 0.857338
-0.277051 0.172291
0.472713 -0.797866
-0.993984 0.548318
-0.712123 -0.684765
-0.028396 -0.272758
-0.286806 0.213103
-0.239420 0.305166
-0.570902 -0.756592
-0.224843 -0.300837
0.836592 -0.339424
0.588059 -0.701938
0.329897 -0.393410
0.665619 -0.207906
0.795020 -0.503768
0.484766 -0.064070
0.264188 0.683340
0.396680 -0.387044
0.495407 -0.926558
0.105932 0.238448
0.892784 0.788921
0.761791 0.239882
-0.332104 -0.091076
0.350134 -0.220503
-0.956813 0.385807
-0.956366 -0.014923
-0.860262 -0.324252
-0.978853 0.848309
0.171874 -0.718525
0.167778 0.614731
-0.527088 -0.438413
0.082229 -0.767793
0.901811 -0.179873
-0.962567 0.977829
0.607381 -0.771085
-0.352736 0.914803
0.555370 -0.904859
-0.802269 -0.917250
-0.551563 0.544991
0.855613 0.255917
-0.740516 0.907067
-0.255860 0.302471
-0.975707 0.917283
0.194564 0.659160
-0.360701 0.618129
0.099892 0.292853
0.155883 0.293180
-0.855034 0.768183
0.875081 0.479298
-0.729573 0.955751
-0.593316 0.613699
0.881973 -0.658700
0.185406 0.851381
0.884976 -0.845481
0.700912 0.437469
0.479148 0.248615
-0.137470 -0.555897
-0.273907 0.746734
-0.385149 -0.908177
-0.197095 -0.818387
-0.589727 0.998359
0.283801 0.683220
0.950554 -0.081220
-0.696724 0.514370
0.313458 0.624100
-0.560598 0.324197
0.907589 -0.074363
0.043486 0.692134
-0.609836 0.289418
0.788961 0.531847
0.736509 0.756250
-0.168949 -0.620253
0.323280 -0.905598
-0.466784 -0.589047
-0.926995 -0.360234
-0.436544 0.516214
0.354740 0.679960
0.928509 -0.509557
0.894601 -0.521909
-0.952347 0.050400
0.567234 -0.587314
0.952248 -0.903653
0.996508 0.632883
0.081097 0.321495
-0.053718 0.028997
0.512072 -0.716421
0.135947 -0.647981
0.738310 0.657229
-0.476077 -0.612139
-0.057306 -0.327382
0.718186 0.340575
-0.955135 0.893285
0.319497 -0.143234
-0.105383 -0.038301
-0.618113 0.588188
0.965144 -0.907132
-0.927688 0.889663
0.530958 0.004177
-0.750277 0.018178
0.952193 0.140935
-0.468121 0.472877
0.757430 -0.087105
-0.493031 -0.128481
0.806408 -0.936953
0.247259 0.710906
0.605844 0.436687
0.985944 -0.802029
0.469455 -0.296440
0.143688 0.158296
0.708412 -0.574250
0.460390 -0.405043
-0.570877 -0.913330
0.988569 0.361830
0.165341 0.087216
