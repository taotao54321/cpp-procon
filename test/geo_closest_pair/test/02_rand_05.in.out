0.0014167999
